playbook.md

```diff
@@ -725,10 +725,27 @@ TODO. This section to cover:
 * Salary negotiation, notice periods and start date
 * Eligibility to work in the UK, documentation that we need
 * Background check
-* Starters checklist
 * The probation period, meeting and expectations
 * ...more things.
 
+#### Joining
+dxw have a robust onboarding process to ensure you feel as supported as possible from your first day.
+
+Pre-day one: You will receive regular communications from our People Manager regarding your first day, these will include discussions on the staff vetting process, your choice of laptop and any upcoming events you may want to put in your diary.
```

Comment:
In this paragraph and the following two, I suggest splitting the sentences with a period rather than a comma, for greater readability.

"You will receive regular communications from our People Manager regarding your first day. These will include discussions on the staff vetting process [...]"